xbmc/addons/AddonManager.h

```diff
@@ -258,6 +268,10 @@ namespace ADDON
     std::set<std::string> m_systemAddons;
     std::set<std::string> m_optionalAddons;
     ADDON_INFO_LIST m_installedAddons;
+
+    // Temporary path where can given to addons, his content becomes during
```

Comment:
```
// Temporary path given to add-ons, whose content is deleted when Kodi is stopped
```

Reply:
Thanks a lot, my bad english is corrected :smile: !